STM32F303K6Tx/Core/Src/main.c

```diff
@@ -60,10 +62,6 @@ uint8_t V_byte[4];
 uint8_t I_byte[4];
 float offset[4] = {0.0, 0.0, 0.0, 0.0};
 uint8_t IDs[2] = {0x30, 0x32};

```

Comment:
I don't think you need these. You should probably use the IDs defined the config.h file in the can lib